_po/ja/how-to/laravel.po

```diff
@@ -768,21 +1175,33 @@ msgid ""
 "}\n"
 "```"
 msgstr ""
+"        for($i =0; $i<2000; $i++)\n"
+"        {\n"
+"            \\App\\Models\\Post::factory()->count(100)->create();\n"
+"        }\n"
+"    }\n"
+"}\n"
+"```"
 
 msgid "### Run migration and seed sample data"
-msgstr ""
+msgstr "### マイグレーション実行とサンプルデータの流し込み"
 
 msgid ""
 "Following command will run the migration to PostgreSQL then seed \n"
 " fake data to it."
 msgstr ""
+"次のコマンドを実行してPostgreSQLにマイグレーションとサンプルデータの流し込みを行います。\n"
 
 msgid ""
 "```shell\n"
 "php arrtisan migrate\n"
 "php artisan db:seed\n"
 "```"
 msgstr ""
+"```shell\n"
+"php arrtisan migrate\n"
+"php artisan db:seed\n"
+"```"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.